lib/strictProxy.js

```diff
@@ -1,11 +1,22 @@
+const meant = require('meant')
+
+
+function didYouMean(scmd, commands) {
```

Comment:
Minor point, but it'd be nice to use an arrow function here since most of the rest of the module uses them